src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -1494,6 +1484,31 @@ void InputProcessor::reportOrphanRecordObjects(EnergyPlusData &state)
             }
         }
 
+        // there are some orphans that we are deeming as special, in that they should be warned in detail even if !DisplayUnusedObjects and
+        // !DisplayAllWarnings
+        if (has_prefix(object_type, "ZoneHVAC:")) {
+            ShowSevereError(state, "Orphaned ZoneHVAC object found.  This was object never referenced in the input, and was not used.");
+            ShowContinueError(state, " -- Object type: " + object_type);
+            ShowContinueError(state, " -- Object name: " + name);
+        } else if (has_prefix(object_type, "Parametric:")) {
```

Comment:
Add warnings for Parametric:\*, GrounHeatTransfer:\*, and HVACTemplate:\* to the orphan (unused) objects report. If these types of objects are present in an incoming epJSON file, they will be ignored, the simulation will run, and these warnings will be generated. If they are present in an incoming idf file, the input processor rejects them and stops with a fatal error.

Reply:
Why the difference in how epJSON and IDF are treated when they contain these objects? Why does epJSON ignore them and continue to run, however, IDF will stop with a fatal error? Shouldn't the functionality be the same between the input formats if they are supposed to be identical input formats currently (aka do the same thing)?

Or is this comment only specific to the ConvertInputFormat program?

Reply:
It probably shouldn't be treated all that much different, but maybe the messaging should be slightly different between epJSON and IDF inputs.  For IDF inputs, yes, you need to just run ExpandObjects and you'll get a new IDF with those objects expanded.  For epJSON inputs, you can't run the IDF-based ExpandObjects, and we don't quite have PyExpandObjects yet for EpJSON files.